<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logician Logic Analyzer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Protocol:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="protocolComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Select a suitable communication protocol.</string>
        </property>
        <item>
         <property name="text">
          <string>General I/O</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I2C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USART</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sample Rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sampleRateComboBox">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <item>
         <property name="text">
          <string>1 MS/s</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Trigger:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="triggerChannelComboBox">
        <item>
         <property name="text">
          <string>Channel 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Channel 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Channel 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Channel 4</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="triggerSlopeComboBox">
        <item>
         <property name="text">
          <string>Rising</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Falling</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="AnalyzerWidget" name="analyzerWidget">
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>50</red>
         <green>50</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_to_Spreadsheet"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_to_Spreadsheet">
   <property name="text">
    <string>Save to Spreadsheet...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalyzerWidget</class>
   <extends>QGraphicsView</extends>
   <header>ui.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
